src/arguments.rs

```diff
@@ -223,6 +224,11 @@ pub fn parse_arguments() -> ArgMatches<'static> {
         .long(Other_commands::token.as_ref())
         .help("Refresh token for authentication.");
 
+    let access_token_arg = Arg::with_name(Other_commands::access_token.as_ref())
+        .takes_value(true)
+        .long(Other_commands::access_token.as_ref())
+        .help("Access token for authentication.");
```

Comment:
```suggestion
	.conflicts_with(Other_commands::access_token.as_ref());
        .help("Access token for authentication.");
```
This arg must conflict the other arg.